airgun/entities/contenthost.py

```diff
@@ -144,7 +133,7 @@ def search_module_stream(self, entity_name, module_name, stream_version=None, st
         view.module_streams.search(query, status)
         return view.module_streams.table.read()
 
-    def install_errata(self, entity_name, errata_id, install_via=None):
+    def install_errata(self, entity_name, errata_id, install_via='rex'):
```

Comment:
Not sure if install_via make would make sense here now :)
```suggestion
    def install_errata(self, entity_name, errata_id):
```

Reply:
It would, it is used on L156 to select one of the two remaining methods - REX with and without customization.